src/hooks/useAuthorities/useAuthorities.js

```diff
@@ -0,0 +1,63 @@
+import { useQuery } from 'react-query';
+import queryString from 'query-string';
+
+import {
+  useOkapiKy,
+  useNamespace,
+} from '@folio/stripes/core';
+
+import { template } from 'lodash';
+
+import { buildQuery } from '../utils';
+
+const AUTHORITIES_API = 'search/authorities';
+
+const useAuthorities = ({ searchQuery, searchIndex, location, history }) => {
```

Comment:
You can remove `location` and `history` from argument in this hook and simply use `useHistory` and `useLocation` hooks here.